chart/develop/templates/cron.yaml

```diff
@@ -0,0 +1,306 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: svc-questionnaire-scheduler-daily-at-one-am
+  namespace: {{ .Values.namespace }}
```

Comment:
All of the resources should not have a namespace. Helm adds the namespace for you with the --namespace flag (which does not come from values.yaml). Idem with ArgoCD -- it gets the namespace from the destination, instead of values.yaml.

Reply:
Sorry if I told you earleir that you could put it in values.yaml :sweat_smile:

Reply:
Ok so I just remove the namespace from everywhere?

Reply:
Fixed